spec/controllers/style_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+describe StyleController do
```

Comment:
Do we even want this style controller anymore? Had a look on the live site, seems like an odd thing to be in an app.

Reply:
I agree, it looks useless.  I'll add it to our new Trello board for tracking app improvements that we want to consider.